anomalib/models/dfkde/normality_model.py

```diff
@@ -15,18 +15,20 @@
 # and limitations under the License.
 
 import random
-from typing import Optional, Tuple
+from typing import List, Optional, Tuple
 
 import torch
+import torchvision
 from torch import Tensor, nn
 
-from anomalib.models.components import PCA, GaussianKDE
+from anomalib.models.components import PCA, FeatureExtractor, GaussianKDE
 
 
 class NormalityModel(nn.Module):
```

Comment:
I would call this `DfkdeModel` to follow the same naming convention as the other algos.